tsconfig.json

```diff
@@ -32,11 +28,5 @@
   "exclude": [
     "build",
     "node_modules"
-  ],
-  "awesomeTypescriptLoaderOptions": {
-    "useWebpackText": true,
-    "useTranspileModule": true,
-    "doTypeCheck": true,
-    "forkChecker": true
-  }
```

Comment:
Nearly all of these options no longer exist, plus configuration for the ts-loader are set in `webpack.config.js` instead.